build.gradle

```diff
@@ -204,6 +204,7 @@ subprojects { project ->
   }
 
   dependencies {
+    //noinspection GradleDynamicVersion
```

Comment:
+ flagged as dynamic version

Reply:
Which `+` ?

Reply:
com.google.errorprone:javac:9+181-r4173-1

Reply:
I can revert and raise a bug against the gradle project I guess?